scripts/version-update.sh

```diff
@@ -32,7 +32,7 @@ var walk = function(dir,done) {
 
 walk("$var", function(err, results) {
     if (err) throw err;
-    var filesToUpdate = results.filter(item => item.includes("package.json")).filter(item => !item.includes("node_modules"));
+    var filesToUpdate = results.filter(item => item.includes("package.json")).filter(item => !item.includes("node_modules") && !item.includes(".next"));
```

Comment:
@jyash97 Instead of just ignoring `.next` folder can you ignore all folders that start with a dot?

Reply:
Fixed. Fiddle to see the logic https://jsfiddle.net/zjv0ue7L/